modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostGISTestSetup.java

```diff
@@ -55,6 +55,8 @@ protected void setUpDataStore(JDBCDataStore dataStore) {
         ((PostGISDialect) dataStore.getSQLDialect()).setLooseBBOXEnabled(false);
 
         // the tests assume non estimated extents
+        // Note: this was due to GEOT-6876, and the tests will now also pass with estimated extents
```

Comment:
How did you come to this factually incorrect, misleading conclusion? 

This line of code has been there since the beginning of the github history, so you did not get it from it.
As you probably noticed, the JDBC tests are largely shared across all databases, thus, bound calculation tests are written to target precise, feature by feature aggregations, which can be performed, one way or the other, in all databases.

Extent _estimation_ enables a trade off, a fast but potentially incorrect calculation (which is fine for common usage, hence it being the default).  For example PostGIS computes it using [ST_Estimated_Extent](http://www.postgis.org/docs/ST_Estimated_Extent.html) which uses statistics to instrantly return a value.
This can result in calculated bounds having different values across databases. Hence, two sets of tests are needed, the shared ones, using a database aggregation function, and the databse specific ones, only for stores that support estimated extents, with results that are unique to them.

Feel free to improve the comment if you want, providing a longer explanation of why estimated extents are turned off, but remove the one you added, a relationship with GEOT-6876 is accidental at best.

Reply:
Thank you @aaime for the great explanation. I apologize for the misunderstanding. I came to that conclusion because all of the PostGIS tests still pass with estimated extents turned on, when the fix is in place.

I think in this instance it's best that I remove the comment, lest I write something else misguided in an attempt to improve it.